src/com/mendix/widget/carousel/components/__tests__/CarouselItem.spec.ts

```diff
@@ -1,23 +1,33 @@
 import { ShallowWrapper, shallow } from "enzyme";
-import { DOM, createElement } from "react";
+import { createElement } from "react";
 
 import { CarouselItem, CarouselItemProps } from "../CarouselItem";
 
 describe("CarouselItem", () => {
     const url = "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7";
-    let wrapper: ShallowWrapper<CarouselItemProps, any>;
-    beforeEach(() => wrapper = shallow(createElement(CarouselItem, { url, active: true })));
+    let output: ShallowWrapper<CarouselItemProps, any>;
 
-    it("should be active", () => {
-        expect(wrapper).toBeElement(
-            DOM.div({ className: "item active" }, DOM.img({ alt: "Carousel image", src: url }))
-        );
+    beforeEach(() => output = shallow(createElement(CarouselItem, { url, active: true })));
```

Comment:
In this case a function is better because you can pass arguments to it. It avoids that you rerender the component like you do on line 23. In that test you actually render the `CarouselItem` twice. That's not pretty.